components/Address.json

```diff
@@ -0,0 +1,53 @@
+{
+  "Address": {
+    "name": "address",
+    "schema": {
+      "$ref": "./Address.json#/address"
+    }
+  },
+  "Addresses": {
+    "name": "addresses",
+    "schema": {"$ref": "./Address.json#/addresses"}
+  },
+  "orNull": {
```

Comment:
When can the addresses be `null`? I avoided this in my rewrite, because I wasn't sure where this was popping up.

Reply:
Also why do we need this in addition to `addressOrNull`?

Reply:
this wasn't used anywhere so I got rid of it